spec_helper.rb

```diff
@@ -12,6 +12,10 @@
   $stdout = File.open("/tmp/fastlane_tests", "w")
 end
 
+# This module is only used to check the environment is currently a testing env
```

Comment:
Why not add it at the very beginning?

Reply:
Because I have noe idea what other strange side effects it could have.

Do you think and advice that I should just move it up to the top?

Reply:
Yeah I think it's a good idea. If ever someone else comes across this file and tries to add something they'll more likely add it below the module definition if it's at the beginning. If it has any adverse effects, we should see tests start to fail

Reply:
Moved it further up, under the requires. Or even above those?

Reply:
Hmm, I think right at the beginning would be best, since other code is already called with the other requires

Reply:
By the way, since I saw you have a merge commit: you can avoid merge commits by doing `git pull --rebase origin master` to rebase your changes on top of the master branch

Reply:
No rebase with the Git web UI ;)